openmc/material.py

```diff
@@ -56,6 +56,9 @@ class Material(object):
         Units used for `density`. Can be one of 'g/cm3', 'g/cc', 'kg/cm3',
         'atom/b-cm', 'atom/cm3', 'sum', or 'macro'.  The 'macro' unit only
         applies in the case of a multi-group calculation.
+    depletable : bool
```

Comment:
Hmmm, could we bring `fissionable` up to the `cross_sections.xml` level?  (the same for any other simple metadata that might be needed).  The trouble with that is right now you can have multiple isotopes/dataset per element in the `cross_sections.xml` file, so this would either have to be an array (one value of fissionable per isotope/dataset), or we dont allow this anymore.